README.md

```
@@ -1,4 +1,5 @@
 
+
```

Markdownlint (0.35.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)